web/html/src/manager/systems/bootstrap/bootstrap-minions.js

```diff
@@ -48,12 +51,41 @@ class BootstrapMinions extends React.Component {
         });
     }
 
+    authMethodChanged(event) {
+        this.setState({
+            authMethod: event.target.value
+        });
+    }
+
     passwordChanged(event) {
         this.setState({
             password: event.target.value
         });
     }
 
+    privKeyFileChanged(event) {
+        const file = event.target.files[0];
```

Comment:
This is not used more than once. Maybe you can just do this inline three lines below
`reader.readAsText(event.target.files[0]`.

Reply:
But many of the `<something>Changed` methods are only used once. I personally find it uglier, to do it all-in once (or maybe I didn't completely got your intention):

```javascript
    privKeyFileChanged(event) {
        this.setState({
            privKeyLoading: true
        });
        const file = event.target.files[0];
        const reader = new FileReader();
        reader.onload = (e) => this.setState({
            privKey: keyString,
            privKeyLoading: false
        });
        reader.readAsText(file);
    }
```

Let me know if this ^ is what you meant.

Reply:
Oh, no no sorry, probably I was not that clear, but it was also not that important.
The `render.onload` is fine as it was, recalling the `this.privKeyLoaded` is fine.

What I meant was only about dropping line
```js
const file = event.target.files[0];
```
because `file` variable is used only once, and just do it at
```js
reader.readAsText(event.target.files[0]);
```

Feel free to keep it or change it, it's not a big deal in the end.

Reply:
I see! Ok, I inlined that `const`.